libpod/runtime_pod_infra_linux.go

```diff
@@ -200,7 +197,15 @@ func (r *Runtime) makeInfraContainer(ctx context.Context, p *Pod, imgName, rawIm
 	if len(p.config.InfraContainer.ConmonPidFile) > 0 {
 		options = append(options, WithConmonPidFile(p.config.InfraContainer.ConmonPidFile))
 	}
-
+	newRes := new(spec.LinuxCPU)
+	if p.config.InfraContainer.CPUPeriod != 0 && p.config.InfraContainer.CPUQuota != 0 {
+		newRes.Period = &p.config.InfraContainer.CPUPeriod
+		newRes.Quota = &p.config.InfraContainer.CPUQuota
+	}
+	if p.config.InfraContainer.ResourceLimits.CPU.Cpus != "" {
```

Comment:
No need for this check since I assume newRes.Cpus by default is ""

Reply:
> No need for this check since I assume newRes.Cpus by default is ""

changing it to `if p.config.InfraContainer.ResourceLimits != nil && p.config.ResourceLimits.CPU != nil` since if you do a simple `podman pod create` ResourceLimits.CPU (I think) will be a nil pointer to a nil pointer